### DISCUSSION TITLE
Cannot do anything with Zowe

### DISCUSSION THREAD

**Discussion by bttrswt** [zowe/zowe-explorer-vscode#2243] [Q&A]
I am currently working on the IBM zxplore and all of a sudden all of my data sets were deleted and I am unable to do anything on Zowe, when I try to add data sets I get this error 

"Error running command zowe.ds.addSession: command 'zowe.ds.addSession' not found. This is likely caused by the extension that contributes zowe.ds.addSession."

What do I need to do?

**Comment by zFernand0** [comment]
hey @bttrswt,
Could you share more details regarding your environment?
- What version of Zowe Explorer are you running?
- How/When did this started to occur? after a VSCode update, or a Zowe Explorer update? 
- Can you share the VSCode version information? 
  Going to  `Help > About` you should get something similar to this
  ![image](https://user-images.githubusercontent.com/37381190/234344508-23a5f332-6785-497e-bd05-262a4b5e9a3d.png)
